src/_packagedcode/pypi.py

```diff
@@ -20,18 +20,16 @@
 from typing import NamedTuple
 import tempfile
 import zipfile
-
 import dparse2
-import packaging
+import packvers as packaging
 import pip_requirements_parser
-import pkginfo2
 from commoncode import fileutils
-from packaging.specifiers import SpecifierSet
+from packvers.specifiers import SpecifierSet
 from packageurl import PackageURL
-from packaging import markers
-from packaging.requirements import Requirement
-from packaging.utils import canonicalize_name
-
+from packvers import markers
+from packvers.requirements import Requirement
+from packvers.utils import canonicalize_name
+import pkginfo2
```

Comment:
See other comment. Do not move this here, leave it above.

Reply:
Done check now.